velox/vector/fuzzer/VectorFuzzer.cpp

```diff
@@ -659,6 +659,19 @@ RowVectorPtr VectorFuzzer::fuzzRowChildrenToLazy(RowVectorPtr rowVector) {
       std::move(children));
 }
 
+SelectivityVector VectorFuzzer::fuzzSelectivity(
```

Comment:
This change seems unrelated. Should it be in a different PR?

Reply:
will move this out in separate PR